db/mongodb/db.go

```diff
@@ -11,27 +11,23 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
-	"go.mongodb.org/mongo-driver/x/network/command"
 	"go.mongodb.org/mongo-driver/x/network/connstring"
 )
 
 const (
-	mongodbUri       = "mongodb.uri"
-	mongodbNamespace = "mongodb.namespace"
-	mongodbAuthdb    = "mongodb.authdb"
-	mongodbUsername  = "mongodb.username"
-	mongodbPassword  = "mongodb.password"
-
-	mongodbUriDefault       = "mongodb://127.0.0.1:27017"
-	mongodbNamespaceDefault = "ycsb.ycsb"
-	mongodbAuthdbDefault    = "admin"
+	mongodbUrl        = "mongodb.url"
+	mongodbAuthdb     = "mongodb.authdb"
+	mongodbUsername   = "mongodb.username"
+	mongodbPassword   = "mongodb.password"
+
+	mongodbUrlDefault        = "mongodb://127.0.0.1:27017/ycsb?w=1"
```

Comment:
We may add some comments here to inform where the default value comes from: https://github.com/brianfrankcooper/YCSB/tree/master/mongodb#mongodb-configuration-parameters